testsuite/features/support/cobbler_test.rb

```diff
@@ -12,18 +12,28 @@
 
 # Class for clobber test
 class CobblerTest
+  ##
+  # It creates a new XMLRPC client object, and then checks to see if the server is running
```

Comment:
```suggestion
  # It creates a new XMLRPC::client object, and then checks to see if the server is running
```
To be consistent with the one above in line 97.